public/server/db.json

```
@@ -169,183 +175,66 @@
         "definition": "Thanks In Advance"
     },
     
-  "lgtm": {
-    "definition": "Looks Good To Me",
-    "alternatives": "Looks Great To Me, Looks Great To Merge"
-  },
-
-  "lfg": {
-    "definition": "Let's Fucking Go",
-    "alternatives": "Life Feels Good, Looking For Group"
-  },
-
-  "idk": {
-    "definition": "I Don't Know"
-  },
-
-  "imo": {
-    "definition": "In My Opinion"
-  },
-
-  "imho": {
-    "definition": "In My Honest Opinion"
-  },
-    
-  "rtfm": {
-    "definition": "Read The Fucking Manual"
-  },
-
-  "gtfo": {
-    "definition": "Get The Fuck Out"
-  },
-
-  "wtf": {
-    "definition": "What The Fuck"
-  },
-
-  "ig": {
-    "definition": "Instagram",
-    "alternatives": "I Guess"
-  },
-
-  "fyi": {
-    "definition": "For Your Information"
-  },
-
-  "iykyk": {
-    "definition": "If You Know You Know"
-  },
-
-  "ifg": {
-    "definition": "I Feel Good"
-  },
-
-  "igtg": {
-    "definition": "I've Got To Go"
-  },
-
-  "icymi": {
-    "definition": "In Case You Missed It"
-  },
-
-  "ikr": {
-    "definition": "I Know, Right?"
-  },
-
-  "kiss": {
-    "definition": "Keep It Simple Stupid",
-    "alternatives": "Keep It Super Simple, Keep It Short And Simple"
-  },
-
-  "hmu": {
-    "definition": "Hit Me Up"
-  },
-
-  "lol": {
-    "definition": "Laugh Out Loud"
-  },
-
-  "stfu": {
-    "definition": "Shut The Fuck Up"
-  },
-
-  "ngl": {
-    "definition": "Not Gonna Lie"
-  },
-
-  "hru": {
-    "definition": "How Are You"
-  },
-
-  "srly": {
-    "definition": "Seriously"
-  },
-
-  "bff": {
-    "definition": "Best Friend Forever"
+  "smh": {
+    "definition": "Shaking My Head"
   },
-
-  "tl": {
-    "definition": "Timeline"
+  
+  "pov": {
+    "definition": "Point Of View"
   },
-
-  "cl": {
-    "definition": "Contact List"
+  
+  "tbf": {
+    "definition": "To Be Frank",
+    "alternatives": "To Be Fair"
   },
-
-  "nvm": {
-    "definition": "Never Mind"
+  
+  "fgs": {
+    "definition": "For God's sake"
   },
 
-  "wyd": {
-    "definition": "What You Doing"
+  "tysm": {
+    "definition": "Thank You So Much"
   },
 
-  "lmk": {
-    "definition": "Let Me Know"
+  "ty": {
+    "definition": "Thank You"
   },
 
-  "idc": {
-    "definition": "I Don't Care"
-  },
+  "syl": {
+    "definition": "See You Later"
+  }, 
 
-  "til": {
-    "definition": "Today I Learn",
-    "alternatives": "Today I Learned"
-  },
+  "hbty": {
+  	"definition": "Happy Birthday To You"
+  }, 
 
-  "brb": {
-    "definition": "Be Right Back"
-  },
+  "hbu": {
+  	"definition": "How About You"
+  }, 
 
-  "byob": {
-    "definition": "Bring Your Own Beer",
-    "alternatives": "Be Your Own Boss"
-  },
+  "wth": {
+  	"definition": "What The Hell"
+  }, 
 
-  "tldr": {
-    "definition": "Too Long, Didn't Read"
-  },
+  "imy": {
+  	"definition": "I Miss You"
+  }, 
 
-  "ttyl": {
-    "definition": "Talk To You Later"
-  },
+  "ilu": {
+  	"definition": "I Love You",
```

Same as the first comment.

Something like the following will work:

```suggestion
  	"definition": "I Love You",
  	"alternatives": "I Like You"
```

---

Got it